webhook_server_container/tests/test_github_api.py

```diff
@@ -0,0 +1,42 @@
+from unittest import mock
```

Comment:
All the tests failed with:

`KeyError: 'repository'`

Reply:
now it failed with `KeyError: 'name'`

`["repository"]["name"]` is mandatory and cannot be empty